samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -10,20 +10,20 @@ Overview
 
 This sample is split into two source files:
 
-* A main file to handle initialization.
-* One additional file for handling Mesh models
+* A :file:`main.c` file to handle initialization.
+* One additional file for handling Mesh models, :file:`model_handler.c`.
 
```

Comment:
@trond-snekvik , can we expand a bit the overview section for this sample?

Reply:
Yes, good idea. I suggest we make this a separate followup task though.